src/applications/disability-benefits/new-686/config/chapters/veteran-information/veteran-address/veteran-address.js

```diff
@@ -0,0 +1,52 @@
+import { genericSchemas } from '../../../generic-schema';
+import emailUI from 'platform/forms-system/src/js/definitions/email';
+
+export const schema = {
+  type: 'object',
+  required: [],
+  properties: {
+    countryDropdown: genericSchemas.countryDropdown,
+    street: genericSchemas.genericTextinput,
+    line2: genericSchemas.genericTextinput,
+    line3: genericSchemas.genericTextinput,
+    city: genericSchemas.genericTextinput,
+    postalCode: genericSchemas.genericNumberInput,
+    phoneNumber: genericSchemas.genericNumberInput,
```

Comment:
We may want to consider changing this to something like `^\\d{10}$` or similar to restrict the amount of numbers they can enter a little more.

Reply:
I love the idea from this and the comment below, although after lot's of testing I could not get the actual regex suggested here to work but I did add a slightly different version of it as a constant and used that in the phoneNumberInput instead